doc/man1/flux-restore.rst

```diff
@@ -54,7 +54,11 @@ OPTIONS
    store.  Performance will vary depending on the content of the archive.
 
 **--size-limit**\ =\ *SIZE*
-   Skip restoring keys that exceed SIZE bytes (default: no limit).
+   Skip restoring keys that exceed SIZE bytes (default: no limit). SIZE
+   may be a floating point number with an optional SI unit 'k' or 'K'
+   for for kibibytes (KiB, units of 1024 bytes), 'M' for mebibytes (MiB,
+   unites of 1024 * 1024 bytes), 'G' for gibibytes (GiB, units of 1024 *
+   1024 * 1024 bytes), and so on up to 'E' for exbibytes (EiB).
```

Comment:
Hm, that's going to be confusing since the letter is actually a 'suffix' in this case. Maybe I'll just drop "suffix" and avoid confusion either way, since it is pretty obvious what is meant? (Or, just do the mention of SI units at all)

Reply:
Or maybe "followed by an SI unit prefix" would work?

Reply:
Maybe like dd(1), "multiplicative suffixes" might be the least confusing?

Reply:
Sounds good to me! I'll also update the PR title. Sorry about that!